src/forward/controller.go

```diff
@@ -56,7 +56,14 @@ func NewController(config *config.Config) (self *Controller, err error) {
 	redisMapper := redisMapper(config).
 		WithInputChannel(joiner.Output)
 
+	appSyncMapper := appSyncMapper(config).
+		WithInputChannel(joiner.Output)
```

Comment:
this won't work. RedisMapper and AppSyncMapper would race for data. You need to put a "duplicator" before both of them.